apps/SageAccountWeb/vite.config.ts

```diff
@@ -5,5 +5,6 @@ export default mergeConfig(vitestConfig, {
   test: {
     include: ['src/tests/**/*.test.[jt]s?(x)'],
     setupFiles: ['src/tests/setupTests.ts'],
+    environment: 'jsdom', // introduced due to random "ReferenceError: window is not defined" during tests
```

Comment:
Wonder why this popped up now! Looks like vite's [default environment](https://vitest.dev/config/#environment) is node